.github/workflows/ci.yaml

```diff
@@ -0,0 +1,71 @@
+name: Docker Image CI
+
+on:
+  push:
+    branches:
+      - '*'
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+  pull_request:
+    branches:
+      - '*'
+      
+jobs:
+
+  lint:
+
+    runs-on: ubuntu-latest
+    steps: 
+    - name: Checkout
+      uses: actions/checkout@master
+    - name: lint
+      uses: luke142367/Docker-Lint-Action@v1.0.0
+      with:
+        target: Dockerfile
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  build:
+
+    runs-on: ubuntu-latest
+
+    #needs: lint
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Get branch name
+      shell: bash
+      run: echo "branch=${GITHUB_HEAD_REF:-${GITHUB_REF#refs/heads/}}" >> $GITHUB_OUTPUT
+      id: branchID
+    - name: Login to Docker Hub
+      uses: docker/login-action@v3
+      with:
+        username: ${{ secrets.DOCKERHUB_USERNAME }}
+        password: ${{ secrets.DOCKERHUB_TOKEN }}
+    - name: Build and push
+      id: docker_build
+      uses: docker/build-push-action@v5
+      with:
+        context: ./
+        file: ./Dockerfile
+        push: true
+        tags: piccio/shell-tools:0.0.${{ github.ref_name }}
```

Comment:
Don't you expect to release any stable tags?

Reply:
yes, I'll do. I've added docker/meta action